client/mass/groups.js

```diff
@@ -44,7 +44,10 @@ class MassGroups {
 			termfilter: appState.termfilter,
 			groups: rebaseGroupFilter(appState),
 			termdbConfig: appState.termdbConfig,
-			customTerms: appState.customTerms
+			customTerms: appState.customTerms,
+			allowedTermTypes: appState.termdbConfig.allowedTermTypes,
+			matrixplots: appState.termdbConfig.matrixplots,
+			vocab: appState.vocab
```

Comment:
thanks a lot for the fix. works great
just a comment  in handlers/samplelst.js line 257, may either replace dofetch with a new vocabApi method, or access genome/dslabel from `app.vocabApi.vocab{genome,dslabel}`
this will remove need to pass state.vocab{} in this line. in general we avoid passing `vocab{}`